requirements.txt

```diff
@@ -1,2 +1,3 @@
 ovos-plugin-manager>=0.0.5
-piper-tts==1.2.0
+piper-tts==1.1.0
```

Comment:
Any reason to not use `1.2.0`?

Reply:
1.2.0 drops support for python < 3.9

Reply:
Python 3.9 is old, that should not be a blocker to move on IMO.

Reply:
If we changed the version constraint to >=1.1.0 would Pip be smart enough to figure out the latest version based on Python version?

Reply:
It should.

```
$ pip install "piper-tts>=1.1.0"
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
Collecting piper-tts>=1.1.0
  Using cached piper_tts-1.2.0-py3-none-any.whl (29 kB)
Requirement already satisfied: piper-phonemize~=1.1.0 in ./.venvs/ovos/lib/python3.11/site-packages (from piper-tts>=1.1.0) (1.1.0)
Requirement already satisfied: onnxruntime<2,>=1.11.0 in ./.venvs/ovos/lib/python3.11/site-packages (from piper-tts>=1.1.0) (1.16.3)
Requirement already satisfied: coloredlogs in ./.venvs/ovos/lib/python3.11/site-packages (from onnxruntime<2,>=1.11.0->piper-tts>=1.1.0) (15.0.1)
Requirement already satisfied: flatbuffers in ./.venvs/ovos/lib/python3.11/site-packages (from onnxruntime<2,>=1.11.0->piper-tts>=1.1.0) (20181003210633)
Requirement already satisfied: numpy>=1.24.2 in ./.venvs/ovos/lib/python3.11/site-packages (from onnxruntime<2,>=1.11.0->piper-tts>=1.1.0) (1.26.3)
Requirement already satisfied: packaging in ./.venvs/ovos/lib/python3.11/site-packages (from onnxruntime<2,>=1.11.0->piper-tts>=1.1.0) (23.2)
Requirement already satisfied: protobuf in ./.venvs/ovos/lib/python3.11/site-packages (from onnxruntime<2,>=1.11.0->piper-tts>=1.1.0) (5.26.0rc1)
Requirement already satisfied: sympy in ./.venvs/ovos/lib/python3.11/site-packages (from onnxruntime<2,>=1.11.0->piper-tts>=1.1.0) (1.12)
Requirement already satisfied: humanfriendly>=9.1 in ./.venvs/ovos/lib/python3.11/site-packages (from coloredlogs->onnxruntime<2,>=1.11.0->piper-tts>=1.1.0) (10.0)
Requirement already satisfied: mpmath>=0.19 in ./.venvs/ovos/lib/python3.11/site-packages (from sympy->onnxruntime<2,>=1.11.0->piper-tts>=1.1.0) (1.3.0)
Installing collected packages: piper-tts
Successfully installed piper-tts-1.2.0
```